docs/docs/contributing.md

```diff
@@ -5,7 +5,43 @@ title: Contributor Guidelines
 
 # Contributor Guidelines
 
-## Getting Started
+## Contributing Documentation Changes
+
+You were confused about our documentation? You ran into a
+pitfall that others also might run into? Help us making the Marathon documentation great.
+ 
+The documentation that is published [here](https://mesosphere.github.io/marathon/) actually gets
+generated from what is found in the docs directory.
+
+If you simply want to correct a spelling mistake or improve the wording of a sentence, you can browse
+the mark down files [here](https://github.com/mesosphere/marathon/tree/master/docs) and use the edit
+button above the markup. That will make it easy to create a pull request that will be reviewed by us.
+
+If want to contribute a larger improvement to our documentation: 
+
+* edit the files in the docs directory
+* check the rendered the result as described in the 
```

Comment:
s/the result/result/

I would capitalise the beginning of this bullet points, or make the following one start with lower case =).